src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
+    private double sideA;
+    private double sideB;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double height, String color) {
+        super.setColor(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.height = height;
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, consider which variables should be local in the method and which should be class-level. In this case, 'color' should be passed to the superclass constructor, not set via a method call.